### DISCUSSION TITLE
How to declare a schema name for each model?

### DISCUSSION THREAD

**Discussion by CursedMun** [prisma/prisma#17343] [Q&A]
How to declare a schema name for each model? in typeorm for example @Entity({schema: 'test'}) so it will use db/test for that specific model instead of db/public

**Comment by nurul3101** [comment]
Hello @CursedMun 👋 

You would need to use the [MultiSchema](https://www.prisma.io/docs/guides/database/multi-schema#how-to-enable-the-multischema-preview-feature) Preview feature. You can then use the `@@schema` attribute to specify the schema name for each model. Let me know in case you run into any issues.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
